### DISCUSSION TITLE
Hamster/Manbino Hoodie

### DISCUSSION THREAD

**Discussion by Telknub** [sunflower-land/sunflower-land#1703] [Art]
This is a Hoodie I made that Manbino, a discord moderator asked me.
I used the hoodies that already in the game as a template to make it.

Previews:
![farmer](https://user-images.githubusercontent.com/38796969/202779612-a7275ba9-cfcb-469c-af5b-4ad57c3bee3a.png)
![manbino](https://user-images.githubusercontent.com/38796969/202779638-df024c74-7f0a-431a-8f1e-c7a0bc98a91c.png)
Hoodie:
![manbino1](https://user-images.githubusercontent.com/38796969/202779643-459d114e-e81d-4d67-bc76-b6848c4e75dc.png)

**Comment by LucasAG95** [comment]
I want mine!

**Reply by MAISAbr** [reply]
AMEIIIII

**Comment by Galphath** [comment]
I want one!

**Comment by aimlemonade** [comment]
Love it! Let's get this in game please! <3

**Comment by SrCouve** [comment]
This is very good, I already want mine

**Comment by Myth727** [comment]
> This is a Hoodie I made that Manbino, a discord moderator asked me. I used the hoodies that already in the game as a template to make it.
> 
> Previews: ![farmer](https://user-images.githubusercontent.com/38796969/202779612-a7275ba9-cfcb-469c-af5b-4ad57c3bee3a.png) ![manbino](https://user-images.githubusercontent.com/38796969/202779638-df024c74-7f0a-431a-8f1e-c7a0bc98a91c.png) Hoodie: ![manbino1](https://user-images.githubusercontent.com/38796969/202779643-459d114e-e81d-4d67-bc76-b6848c4e75dc.png)

Love this all, you inspired me to play around more using pixel art. I used the hoodie with your color scheme. Just went for Christmas style snowman :)

![Untitled 11-19-2022 01-15-11](https://user-images.githubusercontent.com/96928557/203177484-9ea08976-5bf9-413e-b2e3-64530d9771ea.png)

**Comment by adamhannigan** [comment]
Awesome work!

We would need to call this the Manbino as an inside joke ;)

I would suggest trying a lighter border around the hamster to blend into the hoodie 😄

**Reply by Telknub** [reply]
Like this? or even lighter?
![hoodie](https://user-images.githubusercontent.com/38796969/204020110-c8cfd311-48ae-4bc5-8b4e-fc29adf18bdd.png)
![farmer](https://user-images.githubusercontent.com/38796969/204020111-1eb2191e-3d61-4a92-8e77-5f99bcc3083d.png)

**Comment by xSubZero** [comment]
I need this!

**Comment by THINKTRONIK** [comment]
Beautiful 😍

**Comment by 0xiNinja** [comment]
I loved  the  Hoodie's design, a cute  hamster logo  on it and gamma of the colors  - it looks attractive and comfy for the fellow Bumpkin !
Nicely done! @Telknub

**Comment by rodolfoddias** [comment]
We could have a community Island where you can design your own hoodie. All you need is a PNG and some SFL :)
